File: golang/1550.go
func threeConsecutiveOdds(arr []int) bool {
    odd := 0
    for _, val := range arr {
        if val % 2 == 1 {
            odd += 1
            if odd == 3 {
                return true
            }
        } else {
            odd = 0
        }
    }
    return false
}

File: python3/1550.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        odd = 0
        for num in arr:
            if num % 2 == 1:
                odd += 1
                if odd == 3:
                    return True
            else:
                odd = 0
        return False
